keras_hub/src/models/differential_binarization/losses.py

```diff
@@ -0,0 +1,126 @@
+import keras
+from keras import ops
+
+
+class DiceLoss:
+    def __init__(self, eps=1e-6, **kwargs):
```

Comment:
A brief comment explaining these would be helpful.

Reply:
Done.